postinstall/upgrade_many.py

```diff
@@ -263,7 +266,9 @@ def ssh_unchecked_call(*args):
         upgraded_hosts += 1
         if not skip_health_checks:
         # We first want to see the cluster as unhealthy before we wait for it to become healthy
-            wait_for_unhealthy_cluster()
+            if not skip_unhealthy_checks:
+                sleep(10)
```

Comment:
you shouldn't need this sleep. by sleeping here you might _miss_ the unhealthy status (!)

Reply:
Oops😬
This should be the other way around..
Its meant to be in the else